src/react/context/types.ts

```diff
@@ -173,6 +174,11 @@ export interface EscolaLMSContextAPIConfig {
     | API.DefaultResponse<API.CourseProgress>
     | API.DefaultMetaResponse<API.CourseProgress>
   >;
+  fetchMyAuthoredCourses: () => Promise<
+    | void
+    | API.DefaultResponse<API.Course[]>
+    | API.DefaultMetaResponse<API.Course[]>
+  >;
```

Comment:
According to `myAuthoredCourses` method types should be:
```suggestion
  fetchMyAuthoredCourses: () => Promise<API.DefaultResponse<API.Course[]>>;
```